cmd/command.go

```diff
@@ -223,6 +223,39 @@ func rpop(cli *Client, args [][]byte) (interface{}, error) {
 	return popInternal(cli.db, args, false)
 }
 
+func lmove(cli *Client, args [][]byte) (interface{}, error) {
```

Comment:
I think, `lMove` is a more convenient naming for the Go language. You know Go follows the camelCase naming convention. 
```suggestion
func lMove(cli *Client, args [][]byte) (interface{}, error) {
```

Reply:
@gozeloglu 
Yes, "lMove" is better, but for consistency with other existing command function names. Maybe we can update all names in next pr.
https://github.com/flower-corp/rosedb/blob/7b155b6de9816eaad04913c58206dfecd8444f65/cmd/cli.go#L20-L25

Reply:
No worry about that. I've changed the names in my PR: https://github.com/flower-corp/rosedb/pull/152. It is still waiting for merge.

Reply:
@gozeloglu OK